src/compressed_tensors/quantization/lifecycle/initialize.py

```diff
@@ -160,7 +168,9 @@ def _initialize_scale_zero_point(
             num_groups = weight_shape[1] // quantization_args.group_size
             expected_shape = (weight_shape[0], max(num_groups, 1))
 
-    scale_dtype = module.weight.dtype
+    scale_dtype = scale_dtype if scale_dtype is not None else module.weight.dtype
+    # Should we really be setting a case here? More so would indicate that something
+    # is wrong if the condition below is not True
```

Comment:
is this a TODO? seems like something we'd want to address?

Reply:
lol more of a discussion point. I'll turn it into an actionable TODO